### PR TITLE
Updated Flutter platform view demo for iOS Swift

### PR DESCRIPTION
**Pull request by omatt**
**FLNativeViewFactory.swift**
Calling `super.init()` before `_view = UIView()` causes errors, swapped.
Added demo on how [UIView](https://developer.apple.com/documentation/uikit/uiview) can be modified inside `view()` function.

**AppDelegate.swift**
Added unwrap on `registrar! `, and changed function call from `messenger` to `messenger()`
`GeneratedPluginRegistrant.register(with: self)` expected argument is `with` instead of `withRegistry`

**FLPlugin.swift**
Changed function call from `registrar.messenger` to `registrar.messenger()`

Expected output when demo is run on iOS:

Screen Shot 2020-11-05 at 12 20 55 AM

Fixes [#67393](https://github.com/flutter/flutter/issues/67393).

Changes proposed in this pull request:

Proposed changes mentioned above. Copy-pasting this iOS Swift sample should now run without issues on Xcode.

### CODE REVIEW

File: src/docs/development/platform-integration/platform-views.md [hunk 1]
```diff
@@ -444,13 +444,13 @@ import Flutter
 import UIKit
 
 class FLNativeViewFactory: NSObject, FlutterPlatformViewFactory {
-    private weak var messenger: FlutterBinaryMessenger
-
+    private var messenger: FlutterBinaryMessenger
+
```
Comment (blasten) [inline src/docs/development/platform-integration/platform-views.md:448]: nit: remove this and the trailing whitespaces below.

File: src/docs/development/platform-integration/platform-views.md [hunk 2]
```diff
@@ -466,18 +466,28 @@ class FLNativeViewFactory: NSObject, FlutterPlatformViewFactory {
...
     func view() -> UIView {
+        // This function returns a UIView and you can add native iOS UI here.
+        _view.backgroundColor = UIColor.blue
+        
+        let nativeLabel = UILabel()
+        nativeLabel.text = "Native text from iOS"
+        nativeLabel.textColor = UIColor.white
+        nativeLabel.textAlignment = .center
+        nativeLabel.frame = _view.frame
+        _view.addSubview(nativeLabel)
```
Comment (blasten) [inline src/docs/development/platform-integration/platform-views.md:489]: we actually recommend to do this work in the constructor. is there a reason why the getter needs to do this work? cc @bparrishMines

File: src/docs/development/platform-integration/platform-views.md (hunk 1 above)
Comment (omatt) [reply src/docs/development/platform-integration/platform-views.md]: Trailing whitespaces removed

File: src/docs/development/platform-integration/platform-views.md (hunk 2 above)
Comment (omatt) [reply src/docs/development/platform-integration/platform-views.md]: I've tried creating views in the constructor, but for some reason I'm unable to fetch the `UIView.frame` where `UILabel` in the sample will be placed. The output here is that the `UILabel` won't be displayed, but the background color was able to change.

```swift
init(
    frame: CGRect,
    viewIdentifier viewId: Int64,
    arguments args: Any?,
    binaryMessenger messenger: FlutterBinaryMessenger?
) {
    _view = UIView()
    super.init()
    createNativeView(view: _view)
}
```

Moved creating views into a function

```swift
func createNativeView(view _view: UIView){
    _view.backgroundColor = UIColor.blue

    let nativeLabel = UILabel()
    nativeLabel.text = "Native text from iOS"
    nativeLabel.textColor = UIColor.white
    nativeLabel.textAlignment = .center
    nativeLabel.frame = _view.frame
    _view.addSubview(nativeLabel)
}
```

Any idea what might have been the issue? And any suggestions for best practices that we can add in the docs?

Comment (omatt) [reply src/docs/development/platform-integration/platform-views.md]: Thanks! I've moved the creation of views in `init()`

**Comment by sfshaza2** [comment]
Can you review, @johnpryan?

**Comment by johnpryan** [comment]
I'm not sure I'm the right person to review. Maybe @blasten could take a look?

**Comment by blasten** [comment]
LGTM

@jmagman do you mind also reviewing the Swift code?

**Review by blasten — APPROVED** [review]

File: src/docs/development/platform-integration/platform-views.md [hunk 3]
```diff
@@ -473,13 +473,25 @@ class FLNativeView: NSObject, FlutterPlatformView {
         arguments args: Any?,
         binaryMessenger messenger: FlutterBinaryMessenger?
     ) {
-        super.init()
         _view = UIView()
+        super.init()
```
Comment (blasten) [inline src/docs/development/platform-integration/platform-views.md:477]: doesn't this need to be called first?

**Comment by jmagman** [comment]
> @jmagman do you mind also reviewing the Swift code?

I've never written any Swift code.  Maybe @xster?  
There's an internal Swift readability group, maybe we can ask someone there?

File: src/docs/development/platform-integration/platform-views.md (hunk 3 above)
Comment (xster) [reply src/docs/development/platform-integration/platform-views.md:477]: either is fine but I'd just leave this first. We don't expect the super to access any overridden properties.

**Review by xster — APPROVED** [review]

File: src/docs/development/platform-integration/platform-views.md (hunk 3 above)
Comment (omatt) [reply src/docs/development/platform-integration/platform-views.md:477]: Calling `super.init()` first throws this error in Xcode.

Screen Shot 2020-11-21 at 4 42 37 PM

**Pull request merged**